doc/model/train-se-atten.md

```diff
@@ -122,6 +126,16 @@ We highly recommend using the version 2.0 of the attention-based descriptor `"se
       "set_davg_zero": false
 ```
 
+When using PyTorch backend, you must continue to use descriptor `"se_atten"` and specify `tebd_input_mode` as `"strip"` and `smooth_type_embedding` as `"true"`, which achieves the effect of `"se_atten_v2"`. The `tebd_input_mode` can take `"concat"` and `"strip"` as values. When using TensorFlow backend, you need to use descriptor `"se_atten_v2"` and do not need to set `tebd_input_mode` and `smooth_type_embedding` because the default value of `tebd_input_mode` is `"strip"`, and the default value of `smooth_type_embedding` is `"true"` in TensorFlow backend. When `tebd_input_mode` is set to `"strip"`, the embedding matrix $\mathcal{G}^i$ is constructed as:
```

Comment:
When cherry-picking #3828 to `r2`, need to check if this sentence applies to `r2`. Need review from @iProzd

Reply:
In tensorflow, only type_one_side=false is supported when using se_atten_v2, see #3745 . So the formulation here is only the former one. Other sentences about tf is correct.